modules/service/src/main/resources/db/migration/V0030__observations.sql

```diff
@@ -1,3 +1,77 @@
+--- AIR MASS
+
+create domain d_air_mass as numeric(3,2)
+check (value between 1.0 AND 3.0);
+comment on domain d_air_mass is 'air mass limited to the range [1.00, 3.00]';
+
+--- HOUR ANGLE
+
+create domain d_hour_angle as numeric(3,2)
+check (value between -5.0 and 5.0);
+comment on domain d_hour_angle is 'hour angle limited to the range [-5.0, 5.0]';
+
+
+--- CLOUD EXTINCTION
+
+create table t_cloud_extinction (
+  c_tag        d_tag         primary key,
+  c_short_name varchar       not null,
+  c_long_name  varchar       not null,
+  c_value      numeric(2,1)  not null check(c_value > 0.0 and c_value <= 3.0)
+);
+
+insert into t_cloud_extinction values ('PointOne',   '0.1', '0.1', 0.1);
+insert into t_cloud_extinction values ('PointThree', '0.3', '0.3', 0.3);
+insert into t_cloud_extinction values ('PointFive',  '0.5', '0.5', 0.5);
+insert into t_cloud_extinction values ('OnePointZero',   '1.0', '1.0', 1.0);
+insert into t_cloud_extinction values ('OnePointFive',   '1.5', '1.5', 1.5);
+insert into t_cloud_extinction values ('TwoPointZero',   '2.0', '2.0', 2.0);
+insert into t_cloud_extinction values ('ThreePointZero', '3.0', '3.0', 3.0);
+
+--- IMAGE QUALITY
+
+create table t_image_quality (
+  c_tag        d_tag         primary key,
+  c_short_name varchar       not null,
+  c_long_name  varchar       not null,
+  c_value      numeric(2,1)  not null check(c_value > 0.0 and c_value <= 2.0)
+);
+
+insert into t_image_quality values ('PointOne',   '0.1', '0.1', 0.1);
```

Comment:
I'd like to switch these `Point`* values to `ZeroPoint`* to match the others if nobody objects.  Would require updating `lucuma-core` at the moment.